README.md

```diff
@@ -83,6 +83,12 @@ After adding above, add the following dependency definitions.
 
 <!-- OPTIONAL -->
 <dependency>
+    <groupId>org.terasoluna.gfw</groupId>
+    <artifactId>terasoluna-gfw-codepoints</artifactId>
+</dependency>
```

Comment:
@btyoshikawaa 
I removed the descriptions about catalog. 2782cb8ce58269e4578852822c78d6281e0eb7f5.
and arranged the order of `string`,`codepoints`,`validator`. 49af6def209582b432b34602e8f81eea5db7d4d7.